test/groovy/CheckForLegacyConfigurationTest.groovy

```diff
@@ -240,7 +241,7 @@ class LegacyConfigurationCheckUtilsTest extends BasePiperTest {
         String expectedWarning = "[WARNING] Your package.json file package.json contains an npm script using the deprecated name oldNpmScriptName. " +
             "Please rename the script to newNpmScriptName, since the script oldNpmScriptName will not be executed by the pipeline anymore. test"
 
-        LegacyConfigurationCheckUtils.checkForRenamedNpmScripts(nullScript, configChanges)
+        checkForLegacyConfiguration.checkForRenamedNpmScripts(nullScript, configChanges)
```

Comment:
checkForLegacyConfiguration vs stepRule.step.checkForLegacyConfiguration is not clear to me. Why is that used differently in this test?